WordPress/Classes/Networking/WordPressClient.swift

```diff
@@ -45,9 +45,19 @@ actor WordPressClient {
         self.rootUrl = rootUrl.url()
     }
 
-    static func `for`(site: WordPressSite, in session: URLSession) throws -> WordPressClient {
+    static func `for`(site: WordPressSite) throws -> WordPressClient {
```

Comment:
(nit) factory methods should begin with `make`:

> Begin names of factory methods with “make”, e.g. x.makeIterator().

from https://www.swift.org/documentation/api-design-guidelines/

Reply:
That probably doesn't apply here. Because the function is used as

```swift
let client = WordPressClient.for(site)
```

I think `site.makeClient()` makes more sense than `WordPressClient.make(site)`. But I have changed this function to be an initialiser in https://github.com/wordpress-mobile/WordPress-iOS/pull/23715/commits/3343c287b9cbbee92f107864d2bf2a532b4386cb.